ui.frontend/src/main/webpack/site/js/components/buildFacetsGroup.ts

```diff
@@ -24,7 +22,7 @@ const buildFacetsGroup = ({
   searchValue,
   queryParameterName,
   tabId,
-  onSearchItemClick,
+  // onSearchItemClick,
```

Comment:
Please remove comment if not in use anymore

Reply:
This is done